sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/application-changes-report-integration/_tests_/application-changes-report-integration.scheduler.e2e-spec.ts

```diff
@@ -226,16 +228,19 @@ describe(
       const mockedJob = mockBullJob<void>();
 
       // Act
-      const processingResult = await processor.processApplicationChanges(
-        mockedJob.job,
-      );
+      const result = await processor.processQueue(mockedJob.job);
+
       // Assert
       // Assert process result.
-      expect(processingResult).toContain("Process finalized with success.");
-      expect(processingResult).toContain("Applications reported: 0");
+      expect(result).toStrictEqual([
+        "Applications reported: 0",
+        `Uploaded file name: ${expectedFileName}`,
+      ]);
       expect(
         mockedJob.containLogMessages([
+          "Retrieving all application changes which were not reported already.",
```

Comment:
Very minor: maybe it could be rephrased to: "Retrieving all application changes that have not yet been reported."

Reply:
Updated as suggested.